### ISSUE TITLE
Add detailed helptext for the nowplaying command

### ISSUE THREAD

**Issue by za419** [za419/CadenceBot#70]
Add an entry into `default-config.json` under `helpTopics`. The key should probably be 'nowplaying' or possibly 'now playing', and the description should describe how one can use the command and how it will behave from a user perspective.

Question: Should we add a mechanism to define an alias for help topics? That is, a way to config one helptext to act as if another had been used instead...

```json
"helpTopics": {
    "nowplaying": {
        "internalKey": "nowplaying",
        "subtitle": ...,
        "lines": [...],
    },
    "now playing": {
        "alias": "nowplaying"
    }
}
```

And then "Cadence help now playing" automatically translates to "Cadence help nowplaying"?

Just a thought. If we agree this is a good idea, we can split that into another issue.

**Comment by JunhaoSLi** [comment]
In response to question, I don't think it's necessary. Reasoning is that someone without a cs background could probably understand why "Cadence help now playing" doesn't work because the actual command is "nowplaying". Still, I'm not against the feature (I guess I'm abstaining?).

**Comment by za419** [comment]
The wrinkle is that the command, in default config, is in fact "Cadence now playing":

![image](https://user-images.githubusercontent.com/15851480/104065292-1ead1980-51c5-11eb-92cc-fddc3b4ad719.png)


So there's a good argument to be made for why *either* is valid, and therefore for why *both* can be valid - That's my view at least.

**Comment by za419** [comment]
Closed by #86
